python/README.md

```diff
@@ -35,32 +35,32 @@ simulators. The GPU-based simulators included in the CUDA Quantum Python wheels
 require an existing CUDA installation. Additionally, multi-GPU simulators
 require an existing MPI installation.
 
-In most cases, the CUDA and MPI dependencies can be installed via package
-manager. On Ubuntu 22.04, for example, the following commands install all
-optional CUDA dependencies:
+To install both CUDA and CUDA-aware MPI dependencies, we recommend
+installing the [NVIDIA HPC Software Development Kit (SDK)](https://developer.nvidia.com/hpc-sdk).
+Since CUDA Quantum requires CUDA version 11.x,
+please install the SDK bundle with CUDA 11 support.  
+
+On Ubuntu 22.04, for example, the following commands
```

Comment:
```suggestion
On Ubuntu 22.04, the following commands
```
clause not needed

Reply:
revised to use conda instead, which is not OS specific.